catalog/dags/popularity/popularity_refresh_types.py

```diff
@@ -0,0 +1,63 @@
+"""
+# Popularity Refresh DAG Configuration
+This file defines the type for the `PopularityRefresh`, a dataclass containing
+configuration for a Popularity Refresh DAG, and defines the actual
+`POPULARITY_REFRESH_CONFIGS` for each of our media types. This configuration info
+is used to generate the dynamic Popularity Refresh dags.
+"""
+from dataclasses import dataclass, field
+from datetime import datetime, timedelta
+
+
+@dataclass
+class PopularityRefresh:
+    """
+    Configuration object for a popularity refresh DAG.
+
+    Required Constructor Arguments:
+
+    media_type: str describing the media type to be refreshed.
+
+    Optional Constructor Arguments:
+
+    default_args:                      dictionary which is passed to the
+                                       airflow.dag.DAG __init__ method.
+    start_date:                        datetime.datetime giving the
+                                       first valid logical date of the DAG.
+    schedule:                          string giving the schedule on which the DAG
+                                       should be run.  Passed to the
+                                       airflow.dag.DAG __init__ method.
+    refresh_popularity_batch_timeout:  timedelta expressing the amount of time
+                                       refreshing popularity scores for an individual
+                                       batch of records may take.
+    refresh_metrics_timeout:           timedelta expressing amount of time the
+                                       refresh popularity metrics and constants
+                                       may take.
+    doc_md:                            str used for the DAG's documentation markdown
```

Comment:
Is this still accurate?

Reply:
Good call -- I don't think we have a use case for overriding this anyway. Removed.